autoscheduler/frontend/src/components/SelectTerm.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { Button, ClickAwayListener, Grow, Paper, Popper, MenuItem, MenuList,
+  Grid, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'flex',
+  },
+  paper: {
+    marginRight: theme.spacing(2),
+  },
+}));
+
+const style = {
+  width: 650
+}
+
+export default function MenuListComposition() {
+  const classes = useStyles(useStyles);
+  const [open, setOpen] = React.useState(false);
```

Comment:
Nice! You got React Hooks working! I would've expected you to just use class components, but if you understand hooks, that's even better because it looks like that's the direction React is going in the future.

@ryan-conn I'd like for all of our React components to be the same and either stick to class components or stick to function components. I'm personally in favor of function components because a) it's the future of React, b) Christopher's already done them, and c) I think they're cool, but I understand if y'all want class components because they make more intuitive sense. Thoughts?

Reply:
I agree with using function components as well because like you said React is moving towards preferring them over class components, so it's good practice to use them